File: booking-backend/src/main/java/com/stenmartin/project/booking_backend/dal/entity/TireWorkshop.java
package com.stenmartin.project.booking_backend.dal.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Set;

@AllArgsConstructor
@Data
@NoArgsConstructor
public class TireWorkshop {
    private String id;
    private String name;
    private String address;
    private String baseUrl;
    private String apiVersion;
    private Set<VehicleType> supportedVehicleTypes;
}

File: booking-frontend/src/entity/TireWorkshop.ts
export class TireWorkshop {
    id: string;
    name: string;
    address: string;
    supportedVehicleTypes: string[];
  
    constructor(
      id: string = '',
      name: string = '',
      address: string = '',
      supportedVehicleTypes: string[] = []
    ) {
      this.id = id;
      this.name = name;
      this.address = address;
      this.supportedVehicleTypes = supportedVehicleTypes;
    }
  }